Procfile.dev

```diff
@@ -1,3 +1,4 @@
 web: PORT=3000 bundle exec puma -C config/puma.rb
+sidekiq: bundle exec sidekiq -q default -q push -q pull -q mailers
```

Comment:
Is it possible to just do `bundle exec sidekiq` here and have it detect the queues from the newly added config file ... or do we need to specify the queues here?

If we CAN rely on just the config file, we can probably also remove the list of `-q` options from the main `Procfile`